CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [17.2]
+- **NEW FEATURE** Add whole member cookie to initial redux configuration on server 
```

Comment:
I'd like to be more specific here - we're not adding the actual cookie, which is a string, but the object encoded by the cookie.

The deprecation notice should probably be called out in it's own bullet point, e.g.

```md
- **DEPRECATED** `state.config.memberId` should not be used. Reference `state.config.member.id`
   instead, which is guaranteed to be populated
```